src/applications/vaos/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "appName": "VA Online Scheduling",
+  "entryFile": "./vaos-entry",
+  "entryName": "vaos-beta",
+  "rootUrl": "/health-care/vaos-beta",
```

Comment:
I think we should put this under just `/vaos-beta` (or `/vaos`). I think we're less likely to cause some kind of issue with the live health care content that way.